src/js/components/Menu/Menu.js

```diff
@@ -439,7 +439,9 @@ const Menu = forwardRef((props, ref) => {
               background={dropBackground || theme.menu.background}
               {...passThemeFlag}
             >
-              {alignControlMirror === 'top' && align.top === 'top'
+              {alignControlMirror === 'top' &&
+              align.bottom !== 'top' &&
```

Comment:
why was this added?

Reply:
To ensure that when `align.bottom !== 'top'` or `align.top !== 'bottom'` we don't add the controlMirror